system/jlib/jlog.hpp

```diff
@@ -756,6 +756,9 @@ extern jlib_decl const LogMsgJobInfo unknownJob;
 
 extern jlib_decl ILogMsgManager * queryLogMsgManager();
 extern jlib_decl ILogMsgHandler * queryStderrLogMsgHandler();
+#ifdef _CONTAINERIZED
+extern jlib_decl void setupContainerizedLogMsgHandlerFromConfig();
```

Comment:
since it's always by definition from config, and it's rather long method name is it worth the "FromConfig" ?